dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -7768,6 +7768,10 @@ components:
           type: object
           additionalProperties:
             $ref: '#/components/schemas/Alias'
+        collectionEntries:
```

Comment:
Note collectionEntries now in Category and in Collection (but `entries` in Category` goes away)